### PR TITLE
update(resource management): line breaks

### PR DESCRIPTION
**Pull request by NathalieC**
* add the note about line breaks in the page.properties file

### CODE REVIEW

File: modules/ROOT/pages/resource-management.adoc [hunk 1]
```diff
@@ -30,6 +30,12 @@ contentType=layout
 
 If a resource contains both `Index.groovy` and `index.html`, the Groovy class takes precedence.
 
+[NOTE]
+
```
Comment (alachambre) [inline modules/ROOT/pages/resource-management.adoc]:
```suggestion
```

Comment (tbouffard) [inline modules/ROOT/pages/resource-management.adoc]: extra line break
```suggestion
```

File: modules/ROOT/pages/resource-management.adoc [hunk 2]
```diff
@@ -30,6 +30,12 @@ contentType=layout
...
+====
+To insert line breaks in a page.properties file, end the line that preceeds the line break with a "\".
+====
```
Comment (tbouffard) [inline modules/ROOT/pages/resource-management.adoc:43]: I suggest to provide an example of properties source file in the admonition block. The user will be able to better understand what we mean.

File: modules/ROOT/pages/resource-management.adoc [hunk 3]
```diff
@@ -30,6 +30,12 @@ contentType=layout
 
 If a resource contains both `Index.groovy` and `index.html`, the Groovy class takes precedence.
 
+[NOTE]
+
+====
+To insert line breaks in a page.properties file, end the line that preceeds the line break with a "\".
```
Comment (alachambre) [inline modules/ROOT/pages/resource-management.adoc]: I suggest you ton point to the official properties javadoc so users will have the fully detailed explanation, with all the important things to take into account to create 'advanced' properties file. 
I suggest you to put it in a TIP admonition, with a content like 

"To write advanced properties file (line break, natural lines, logical lines...) please refer to the official documentation -> https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Properties.html#load(java.io.Reader)

**Review by alachambre — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/resource-management.adoc (hunk 3 above)
Comment (tbouffard) [inline modules/ROOT/pages/resource-management.adoc]: Typo + Remove the double quote: they must be not added in the file
```suggestion
To insert line breaks in a page.properties file, end the line that precedes the line break with a `\`.
```

Comment (tbouffard) [reply modules/ROOT/pages/resource-management.adoc]: Current rendering
image

Comment (alachambre) [reply modules/ROOT/pages/resource-management.adoc]: @NathalieC What about this pull request? 
If you don't want to take it account suggestions let us know.

File: modules/ROOT/pages/resource-management.adoc (hunk 2 above)
Comment (NathalieC) [reply modules/ROOT/pages/resource-management.adoc:43]: ok. @mariellebonita ; can you help by providing an example following the case you have solved on this very topic? That would be great.

Comment (marielle-spiteri) [reply modules/ROOT/pages/resource-management.adoc:43]: example added

**Review by marielle-spiteri — COMMENTED** [review]
example added

File: modules/ROOT/pages/resource-management.adoc (hunk 3 above)
Comment (alachambre) [reply modules/ROOT/pages/resource-management.adoc]: @NathalieC  UUUUPP :)

**Comment by alachambre** [comment]
@NathalieC Don't forget your pull request please :)

File: modules/ROOT/pages/resource-management.adoc (hunk 3 above)
Comment (NathalieC) [reply modules/ROOT/pages/resource-management.adoc]: Here I am. Coming.

File: modules/ROOT/pages/resource-management.adoc (hunk 2 above)
Comment (NathalieC) [reply modules/ROOT/pages/resource-management.adoc:43]: Thanks Marielle!

File: modules/ROOT/pages/resource-management.adoc (hunk 3 above)
Comment (NathalieC) [reply modules/ROOT/pages/resource-management.adoc]: I've added what I think to be the best of both worlds: a quick tip for the use case we know is an issue (line breaks) + the official link to the documentation, to deepen one's knowledge on properties.

**Review by alachambre — APPROVED** [review]

**Pull request merged**